graphql/resolvers.go

```diff
@@ -495,6 +495,13 @@ func (r *mutationResolver) EditSpawnHost(ctx context.Context, editSpawnHostInput
 		opts.DeleteInstanceTags = deletedTags
 	}
 	if editSpawnHostInput.Volume != nil {
+		v, err := r.sc.FindVolumeById(*editSpawnHostInput.Volume)
+		if err != nil {
+			return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Error finding requested volume id: %s", err))
+		}
+		if v.AvailabilityZone != h.Zone {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error mounting volume to spawn host, They must be in the same availability zone."))
```

Comment:
this seems like it would be a BadRequest code

Reply:
Thanks Just replaced it with the closest error type we have which is `InputValidationError`